[EDU-6435] feat: add release notes for Edge AI (#1613)
aziontech/docs: src/content/docs/pt-br/pages/menu-principal/release-notes/release-notes.mdx

---
import Tag from 'primevue/tag'

## 5 de maio, 2025

### Edge AI

> A Azion anuncia seu mais recente produto, **Edge AI**. Ele oferece uma gama de soluções que transformam como as aplicações inteligentes são construídas e entregues.

Reviewer: ```suggestion
A Azion anuncia seu mais novo produto, **Edge AI**, oferecendo uma gama de soluções que transformam como as aplicações inteligentes são construídas e entregues.
```

---
import Tag from 'primevue/tag'

## 5 de maio, 2025

### Novos recursos de AI

A Azion apresenta vários novos recursos de Inteligência Artificial, incluindo **Edge AI**, que permite a execução de modelos de AI avançados diretamente na infraestrutura da Azion, garantindo mínima latência e máximo desempenho.

#### Edge AI

> - **Execução de Modelos de AI:** suporta modelos LLMs, VLMs, multimodais, embedding e reranking no **Edge Runtime**.

Reviewer: ```suggestion
- **Execução de modelos de AI:** suporta modelos LLMs, VLMs, multimodais, embedding e reranking no **Edge Runtime**.
```

> - **Execução de Modelos de IA**: Suporta modelos LLMs, VLMs, multimodais, embedding e reranking no **Edge Runtime**.
> - **Personalização de Modelos**: Ajuste fino de modelos com **Low-Rank Adaptation (LoRA)** para necessidades específicas de negócios.

Reviewer: O Gabriel sugeriu manter fine-tuning em inglês na página de produto, acho válido padronizarmos em todas as docs em vez de "Ajuste fino".
```suggestion
- **Execução de Modelos de AI:** suporta modelos LLMs, VLMs, multimodais, embedding e reranking no **Edge Runtime**.
- **Personalização de modelos:** fine-tuning de modelos com **Low-Rank Adaptation (LoRA)** para necessidades específicas de negócios.
```

#### Edge SQL

- **Busca Vetorial:** bancos de dados vetoriais distribuídos e pipelines de RAG eficientes.
- **Integração com LangChain Vector Store:** permite o armazenamento e a recuperação eficientes de documentos vetoriais.
- **Integração com LangChain Vector Retriever:** habilita capacidades avançadas de busca híbrida para documentos, combinando as vantagens de diferentes métodos de busca para resultados mais precisos.

#### Edge Functions

- **Suporte a agentes com LangGraph:** crie agentes de AI com **Tool Calling** e **Agentic RAG**, utilizando o framework LangGraph da [LangChain](https://www.langchain.com/).
- **LangGraph AI Agent Boilerplate:** template para criar um agente que pode consultar e interagir com dados armazenados em bancos de dados EdgeSQL.
- **Compatibilidade:** integração com APIs compatíveis com OpenAI e LangChain/LangGraph para automação de workflows.

> Para mais detalhes, visite a documentação do [Edge AI](/pt-br/documentacao/produtos/edge-ai/), [Edge Functions](/pt-br/documentacao/produtos/build/edge-application/edge-functions/), e [Edge SQL](/pt-br/documentacao/produtos/store/edge-sql/).

Reviewer: ```suggestion
Para mais detalhes, visite a documentação do [Edge AI](/pt-br/documentacao/produtos/edge-ai/), [Edge Functions](/pt-br/documentacao/produtos/build/edge-application/edge-functions/) e [Edge SQL](/pt-br/documentacao/produtos/store/edge-sql/).
```
